src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/HigherEducationFunction.xml

```diff
@@ -1,51 +1,42 @@
-	<bean id="HigherEducationFunction" parent="HigherEducationFunction-parentBean">
-		<property name="attributes">
-			<list merge="true">
-				<ref
-					bean="HigherEducationFunction-extension-financialHigherEdFunctionDescription" />
-			</list>
-		</property>
-	</bean>
+    <bean id="HigherEducationFunction" parent="HigherEducationFunction-parentBean">
+        <property name="attributes">
+            <list merge="true">
+                <ref bean="HigherEducationFunction-extension-financialHigherEdFunctionDescription" />
+            </list>
+        </property>
+    </bean>
 
-	<!-- Attribute Definitions -->
+    <!-- Attribute Definitions -->
 
-	<bean
-		id="HigherEducationFunction-extension-financialHigherEdFunctionDescription"
-		parent="HigherEducationFunction-extension-financialHigherEdFunctionDescription-parentBean" />
-	<bean
-		id="HigherEducationFunction-extension-financialHigherEdFunctionDescription-parentBean"
-		abstract="true"
-		parent="HigherEducationFunctionExtendedAttribute-financialHigherEdFunctionDescription"
-		p:name="extension.financialHigherEdFunctionDescription" />
+    <bean
+        id="HigherEducationFunction-extension-financialHigherEdFunctionDescription"
+        parent="HigherEducationFunction-extension-financialHigherEdFunctionDescription-parentBean" />
+    <bean
+        id="HigherEducationFunction-extension-financialHigherEdFunctionDescription-parentBean"
+        abstract="true"
+        parent="HigherEducationFunctionExtendedAttribute-financialHigherEdFunctionDescription"
+        p:name="extension.financialHigherEdFunctionDescription" />
 
-	<!-- Business Object Inquiry Definition -->
+    <!-- Business Object Inquiry Definition -->
 
-	<bean id="HigherEducationFunction-inquiryDefinition-override"
-		parent="DataDictionaryBeanOverride">
-		<property name="beanName" value="HigherEducationFunction-inquiryDefinition" />
-		<property name="fieldOverrides">
-			<list>
-				<bean parent="FieldOverrideForListElementInsert">
-					<property name="propertyName" value="inquirySections[0].inquiryFields" />
-					<property name="propertyNameForElementCompare" value="attributeName" />
-					<property name="element">
-						<bean parent="FieldDefinition" p:attributeName="financialHigherEdFunctionNm" />
-					</property>
-					<property name="insertAfter">
-						<list>
-							<bean parent="FieldDefinition"
-								p:attributeName="extension.financialHigherEdFunctionDescription" />
-						</list>
-					</property>
-				</bean>
-			</list>
-		</property>
-	</bean>
+    <bean id="HigherEducationFunction-inquiryDefinition-override"
+        parent="DataDictionaryBeanOverride">
+        <property name="beanName" value="HigherEducationFunction-inquiryDefinition" />
+        <property name="fieldOverrides">
+            <list>
+                <bean parent="FieldOverrideForListElementInsert"
+                      p:propertyName="sections[0].fields"

```

Comment:
It seems that this field override has been updated to only affect the new section field list. I don't see any new beans that perform a field override on the legacy section field list. Can you please confirm whether the older-style HigherEducationFunction inquiry is still openable? If it is, then we'll still need a field override that affects the legacy field list.